vhd/packer/configure-windows-vhd-phase2.ps1

```diff
@@ -48,7 +48,7 @@ function Get-ContainerImages {
     # CSE will configure and register containerd as a service at deployment time
     Start-Job -Name containerd -ScriptBlock { containerd.exe }
     foreach ($image in $imagesToPull) {
-        & ctr.exe -n k8s.io images pull $image >> $containerdImagePullNotesFilePath
+        & ctr.exe -n k8s.io images pull $image > $containerdImagePullNotesFilePath

```

Comment:
why? in `bash` this would replace instead of appending, what will `powershell` do?

Reply:
You are correct, this is replacing it each time. Appending each time results in a file that takes far too much storage of the Windows VM. Proper fix would be to ignore the output, but my quick fix did not work. So just sticking with replacing it for now.